readme.md

```diff
@@ -51,6 +52,10 @@ The built-in color picker supports plugins:
 
 ## FAQ
 
+#### What is LCH color?
+
+[It's a more human-friendly color format.](https://lea.verou.me/2020/04/lch-colors-in-css-what-why-and-how/)
+
 #### The color changes if I copy and then paste it
 
 That is because the default color space in the picker is [Display P3](https://en.wikipedia.org/wiki/DCI-P3), but the color space used for the CSS color formats is [sRGB](https://en.wikipedia.org/wiki/SRGB) (browsers do not yet [handle color spaces](https://css-tricks.com/the-expanding-gamut-of-color-on-the-web/)).
```

Comment:
The comment is not really correct. I suggest 

That is because the default color space in the picker is [Display P3](https://en.wikipedia.org/wiki/DCI-P3), which is [part of CSS Color 4](https://drafts.csswg.org/css-color-4/#valdef-color-display-p3), but the color space used for the legacy  CSS color formats is [sRGB](https://en.wikipedia.org/wiki/SRGB) (browsers are starting to [handle color spaces](https://css-tricks.com/the-expanding-gamut-of-color-on-the-web/) but they are not all there yet).

Reply:
More generally though, the picker should be configurable to output either display-p3, or sRGB

Reply:
> More generally though, the picker should be configurable to output either display-p3, or sRGB

I'm not sure it's worth doing this generally until there are more certainly about how all this will work. For example, I see no way to do high gamut HSL.

It might be better for now to just add a setting of whether or not to clamp LCH?

I have a strong feeling LCH will be the main way to specify colors in the future.

Reply:
Continued in https://github.com/sindresorhus/System-Color-Picker/issues/13. I don't want this blocking adding LCH support.

Reply:
> The comment is not really correct. I suggest

Thanks. Changed locally.